src/machinetalk/application/commandbase.cpp

```diff
@@ -353,7 +353,7 @@ void CommandBase::sendEmcAxisUnhome(Container &tx)
 
 void CommandBase::sendShutdown(Container &tx)
 {
-    sendCommandMessage(MT_SHUTDOWN, tx);
+    sendCommandMessage(MT_LAUNCHER_SHUTDOWN, tx);
```

Comment:
This is not part of upstream Machinekit. System shutdown belongs to the mklauncher, see: https://github.com/machinekit/machinekit/blob/master/src/machinetalk/mklauncher/mklauncher.py#L451